draft-ietf-mls-protocol.md

```diff
@@ -862,68 +862,32 @@ the same as the signature algorithm specified in the credential field of the
 KeyPackage objects in the leaves of the tree (including the InitKeys
 used to add new members).
 
-The ciphersuites are defined in section {{mls-ciphersuites}}.
+For all ciphersuites defined in this document, the `Derive-Key-Pair` function
```

Comment:
I think it would make more sense for the ciphersuite / HPKE to define how it generates a private key, given a seed, than to specify an Unmarshal function and do rejection sampling. Rejection sampling isn't guaranteed to work for many schemes -- RSA being the one that comes to mind, but possibly some PQ schemes also.

Reply:
This is a great point. In this case, `Derive-Key-Pair` should just be HPKE's version of `Derive-Key-Pair`, possibly after some HKDF step. This also lets us avoid doing an `Expand-Label` for every iteration.